### PR TITLE
Update ScaleHandle.js

### PR DESCRIPTION
**Pull request by leumasxxx**
Fixes #0000 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [ x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updates
* [@TildaDares  ] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: src/edit/handles/ScaleHandle.js [hunk 1]
```diff
@@ -11,24 +11,24 @@ L.ScaleHandle = L.EditHandle.extend({
...
-    var corner1 = map.latLngToLayerPoint(overlay.getCorner(0));
-    var corner2 = map.latLngToLayerPoint(overlay.getCorner(1));
-    var w = Math.abs(corner1.x - corner2.x);
-    var h = Math.abs(corner1.y - corner2.y);
-    var distance = Math.sqrt(w * w + h * h);
+    const corner1 = map.latLngToLayerPoint(overlay.getCorner(0));
+    const corner2 = map.latLngToLayerPoint(overlay.getCorner(1));
+    let w = Math.abs(corner1.x - corner2.x);
+    let h = Math.abs(corner1.y - corner2.y);
+    cpnst distance = Math.sqrt(w * w + h * h);
```
Comment (TildaDares) [inline src/edit/handles/ScaleHandle.js:31]:
```suggestion
    const distance = Math.sqrt(w * w + h * h);
```

Comment (leumasxxx) [reply src/edit/handles/ScaleHandle.js:31]: Thank you @TildaDares  for editing this I didn’t notice that I had made a typographical error

Comment (TildaDares) [reply src/edit/handles/ScaleHandle.js:31]: Hi @leumasxxx, I haven't changed it yet. You can click the `Commit Suggestion` button to do that. Thanks!

**Review by TildaDares — APPROVED** [review]
Amazing work @leumasxxx. Thank you!

**Comment by TildaDares** [comment]
Congratulations on merging your PR @leumasxxx 🎉🎉

**Comment by leumasxxx** [comment]
> Congratulations on merging your PR @leumasxxx 🎉🎉

@TildaDares thank you

**Pull request merged**